src/rpcblockchain.cpp

```diff
@@ -777,6 +781,16 @@ UniValue getchaintips(const UniValue& params, bool fHelp)
     return res;
 }
 
+UniValue mempoolInfoToJSON()
+{
+    UniValue ret(UniValue::VOBJ);
+    ret.push_back(Pair("size", (int64_t) mempool.size()));
+    ret.push_back(Pair("bytes", (int64_t) mempool.GetTotalTxSize()));
+    ret.push_back(Pair("usage", (int64_t) mempool.DynamicMemoryUsage()));
```

Comment:
`size()` and `GetTotalTxSize()` take the lock themselves.  They return unsigned long and uint64_t.

`DynamicMemoryUsage()` does not take the lock.  It returns a size_t.  The other `Get...()` methods which return a size_t also don't take a lock.

I'm inclined to leave as is for now, since not critical.

It would be better though if all the calls above were under a single lock so the information returned to the user is based on a single snapshot of the mempool.  Maybe something for a future PR.